pkg/virt-api/api.go

```diff
@@ -269,6 +269,21 @@ func (app *virtAPIApp) composeSubresources() {
 			Operation(version.Version + "VNC").
 			Doc("Open a websocket connection to connect to VNC on the specified VirtualMachineInstance."))
 
+		subws.Route(subws.GET(rest.ResourcePath(subresourcesvmiGVR) + rest.SubResourcePath("ssh") + rest.PortPath).
```

Comment:
In general all options including VNC and Console could be `CONNECT` instead of `GET`, but that would break the API and if I only do it for SSH the client logic would be more complicated, so I reverted that an am gonna stick with `GET` for now.